Server/Server.Core/Server.Core.Static/KeysValidator.hpp

```diff
@@ -26,44 +33,53 @@ class KeysValidator
 
 public:
     const Keys keys;
-
+    /**
+     * @brief It's default noexcept constructor
+     * @details Used to initialize vectors of valid keys
+     */
     KeysValidator() noexcept
     {
-        validKeys.emplace_back(keys.realDB);
+        validKeys.emplace_back(keys.listenedPort);
         validKeys.emplace_back(keys.fileDB);
 
         keysWithoutValues.emplace_back(keys.fileDB);
     }
-
-    KeysValidator(const initList& validkeysList, const initList& keysWithoutValuesList) noexcept
-        : KeysValidator()
+    /**
+     * @brief It's constructor with initList
+     * @details Used to initialize vectors of valid keys
```

Comment:
Fixed

Reply:
@brief The custom constructor initialized by extra arguments 
@details In case we want to use some extra arguments and don't want to store that argument in the keyValidator list, we can use this constructor. Exception safe method. Calls default constructor inside.
@param const initList& validKeysList - list of arguments that don't need additional value, unary argument
               const initList& keysWithoutValuesList - list of arguments that need additional value